openassessment/xblock/static/js/src/lms/oa_peer.js

```diff
@@ -26,14 +28,28 @@ OpenAssessment.PeerView.prototype = {
     load: function(usageID) {
         var view = this;
         var stepID = ".step--peer-assessment";
+
+        view.isRendering = true;
         this.server.render('peer_assessment').done(
             function(html) {
```

Comment:
The duplication between the various JS classes is painful. Is there any way to create a shared "done" handler?